src/bridge_lcd.cpp

```diff
@@ -131,7 +140,11 @@ void bridge_lcd::display_wifi_connect_screen(String ap_name, String ap_pass) {
 void bridge_lcd::display_wifi_success_screen(const String& mdns_url, const String& ip_address_url) {
     // This screen is displayed at startup when the TiltBridge is configured to connect to WiFi
     clear();
+#ifdef LCD_TFT_ESPI
+    print_line("Access TiltBridge at:", "", 1);
```

Comment:
I might come back later and see if I can shrink the text to get this to fit (similar to the OLED screens), but yeah - this is the best compromise for the time being.